### PR TITLE
Edit to overlay ref

### PR DESCRIPTION
**Pull request by ratscrew**
have attach return the component

### CODE REVIEW

**Comment by ratscrew** [comment]
I signed it!

File: src/core/overlay/overlay-ref.ts [hunk 1]
```diff
@@ -12,8 +12,9 @@ export class OverlayRef implements PortalHost {
       private _state: OverlayState) { }
 
   attach(portal: Portal<any>): Promise<any> {
-    return this._portalHost.attach(portal).then(() => {
+    return this._portalHost.attach(portal).then((_comp) => {
       this._updatePosition();
+      return _comp;
```
Comment (DevVersion) [inline src/core/overlay/overlay-ref.ts]: I can't see the point of that change. 

The resolved value will be automatically returned, so there is no need to pass it as an attribute and return it "again".

**Comment by ratscrew** [comment]
@DevVersion the component is not being returned without the line, others are having the same issue: #113

**Comment by DevVersion** [comment]
@ratscrew Yeah, just took a look at the issue and now it makes sense.

**Comment by jelbourn** [comment]
I also fixed this behavior in #761, closing this in favor of that one (which should be merging very soon).

**Pull request closed without merging**